cmd/katalyst-agent/app/options/global/base.go

```diff
@@ -52,6 +52,7 @@ type BaseOptions struct {
 
 	ReclaimRelativeRootCgroupPath string
 	GeneralRelativeCgroupPaths    []string
+	OptionalRelativeCgroupPaths   []string
```

Comment:
why options are defined in `BaseOptions` but configs are defined in `MalachiteConfiguration`?